lib/function/miscellaneous

```diff
@@ -217,8 +211,7 @@ if [[ -f "/root/userscripts/uscripts" ]]; then
 		echo -e "\033[1mDone\e[0m."
 	fi
 fi
-if [[ -e "/root/userscripts" ]]; then rm -fdr /root/userscripts; fi
-}
+rm -fdr /root/userscripts; fi
```

Comment:
This appears to break a function and you left an `; fi` which would no longer be required.

Reply:
Fixed.